azerothcore/azerothcore-wotlk
# Meridith gossip menu and texts

## Meridith gossip fix

##### CHANGES PROPOSED:

- New gossip conditions


###### ISSUES ADDRESSED:


Closes #545 


##### TESTS PERFORMED:


Tested in-game

##### HOW TO TEST THE CHANGES:


1. .go creature 40530
2. Talk and check dialog
3. .quest add 8599
4. .quest complete 8599
5. .quest reward 8599
6. Talk to Meridith to confirm new dialog
7. Activate Gossip option to get speed buff

##### KNOWN ISSUES AND TODO LIST:


- [ ] Spell is missing the effect text [Siren's Song](https://db.rising-gods.de/?spell=25678)


##### Target branch(es):

Master
